### DISCUSSION TITLE
commit

### DISCUSSION THREAD

**Discussion by zfast9002** [community/community#54979] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i want to make a commit

**Comment by tomasdemonasterio** [comment, marked as answer]
1. Make changes to the files in your local repository. You can use a text editor or an integrated development environment (IDE) to make these changes.
2. 
3. Once you've made your changes, use the git add command to stage the changes. For example, if you want to stage all changes, you can use the command git add . (note the dot at the end).
4. 
5. After staging your changes, you can use the git commit command to create a new commit. You should include a descriptive message that explains the changes you've made. For example, you can use the command git commit -m "Added a new feature to the website.".
6. 
7. Finally, you can use the git push command to push your changes to the remote repository.

**Reply by Mrarahuman** [reply]
soreji5395@adtolls.com
